### ISSUE TITLE
replicating results from article (CIFAR-10)

### ISSUE THREAD

**Issue by wmpauli** [samet-akcay/skip-ganomaly#12]
Thank you for sharing the code from your article here, I really appreciate it!

I have been trying to replicate the results reported in your article, but to no avail: E.g. for automobiles, the AUC starts out around .85 from the first epoch, and seems to randomly fluctuate around that value, but never goes up from there. I don't see a relationship between any of the losses to the AUC either.

I would be grateful for any help.

I noticed that all hyper parameters (except for the weight for w_con) are as in the article.

I noticed that there is code for a learning rate scheduler, but it doesn't seem to be used. Since it is also not mentioned in the article, I assume this is not the reason I am unable to achieve the same performance.

The one item I could think of is that AUC depends on the stopping criterion. The article states that "we save the parameters of the network when the performance of  the  model  starts  to  decrease  since  this  reduce  is  a  strong indication  of  over-fitting." When I train the model, err_g, err_g_lat, err_g_con continue to decrease beyond 15 epochs, while err_d and err_g_adv don't decrease after the first epoch. Is this pattern expected? Based on which performance did you decide to stop training?

**Comment by banana47** [comment]
Do you replicate the result?

when replicating the result, I have some problems. Some categories's AUC is high in the first fewer epoch, but  when I keep training, the AUC is decrease, do you have the same problems?

There is a max AUC in each category,  do you use the final epoch AUC or the max AUC in each category?In the begaining , I use the final AUC in each category, but the best average result is 0.58. If use the max AUC, the average result is 0.72, but this
will use different epoch to get result, how do you do it?
